pkg/operator/apiserver/controller/nsfinalizer/finalizer_controller.go

```diff
@@ -53,7 +53,10 @@ func NewFinalizerController(
 	return factory.New().ResyncEvery(time.Minute).WithSync(c.sync).WithInformers(
 		kubeInformersForTargetNamespace.Core().V1().Pods().Informer(),
 		kubeInformersForTargetNamespace.Apps().V1().DaemonSets().Informer(),
-	).ToController(fullname, eventRecorder.WithComponentSuffix("finalizer-controller"))
+	).ToController(
```

Comment:
why? the controller doesn't use WithSyncDegradedOnError which is required for reporting a degraded condition on sync err, right ?